netjsonconfig/backends/openwrt/converters/zerotier.py

```diff
@@ -8,20 +8,55 @@ class ZeroTier(OpenWrtConverter, BaseZeroTier):
     _schema = schema['properties']['zerotier']['items']
 
     def __intermediate_vpn(self, vpn):
+        nwid_ifnames = vpn.get('networks', [])
+        files = self.netjson.get('files', [])
+        self.netjson['files'] = self.__get_zt_ifname_files(vpn, files)
         vpn.update(
             {
                 '.name': self._get_uci_name(vpn.pop('name')),
                 '.type': 'zerotier',
-                'join': vpn.pop('id'),
+                'config_path': vpn.get('config_path', '/etc/openwisp/zerotier'),
+                'copy_config_path': vpn.get('copy_config_path', '1'),
+                'join': [networks.get('id', '') for networks in nwid_ifnames],
                 'enabled': not vpn.pop('disabled', False),
             }
         )
+        del vpn['networks']
         return super().__intermediate_vpn(vpn, remove=[''])
 
     def __netjson_vpn(self, vpn):
-        vpn['id'] = vpn.pop('join')
+        nwids = vpn.pop('join')
         vpn['name'] = vpn.pop('.name')
+        vpn['networks'] = [{"id": nwid, "ifname": f"owzt{nwid[-6:]}"} for nwid in nwids]
         # 'disabled' defaults to False in OpenWRT
         vpn['disabled'] = vpn.pop('enabled', '0') == '0'
         del vpn['.type']
         return super().__netjson_vpn(vpn)
+
+    def __get_zt_ifname_files(self, vpn, files):
+        config_path = vpn.get('config_path', '/etc/openwisp/zerotier')
+        nwid_ifnames = vpn.get('networks', [])
+        zt_file_contents = '# network_id=interface_name\n'
+
+        for networks in nwid_ifnames:
+            nwid = networks.get('id', '')
+            ifname = networks.get('ifname', f'owzt{nwid[-6:]}')
```

Comment:
@Aryamanz29 can you please point me to the test case that verifies the fallback `ifname` is used when the user does not provide the `ifname`?